support/cas-server-support-saml-idp/src/main/java/org/apereo/cas/config/SamlIdPEndpointsConfiguration.java

```diff
@@ -576,5 +576,92 @@ public SamlProfileHandlerConfigurationContext samlProfileHandlerConfigurationCon
                 .samlFaultResponseBuilder(samlProfileSamlAttributeQueryFaultResponseBuilder)
                 .build();
         }
+
+        @Bean
+        @Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+        public SamlProfileHandlerConfigurationContext samlIdPInitiatedProfileHandlerConfigurationContext(
```

Comment:
Please remove this bean entirely, and either set the `samlIdPObjectSignatureValidator` on the context class directly when it's built, or set the field when you build the `idpInitiatedSamlProfileHandlerController` bean.

Reply:
Both PRs have been updated to take into account your feedback.